NOTICE.txt

```diff
@@ -1,5 +0,0 @@
-Elastic package-registry
```

Comment:
Is this file removed on purpose?

Reply:
Yes, as there is also LICENSE file: https://github.com/elastic/package-registry/blob/master/LICENSE.txt

Reply:
For example in Beats repo we have both files and I remember there was a reason for this. Not sure if we still need both? @urso you might remember?

Reply:
I see, let's clarify if we need this file. It isn't present in any package related repo (integrations, elastic-package, package-spec, etc.)